production/kubernetes/install-loki.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+# shellcheck shell=bash
```

Comment:
Does it make sense (perhaps as a different issue) to convert these to be plain shell and not rely on any bash-ism?

Reply:
IMO it makes sense to do that, but I think it should be a separate PR